### ISSUE TITLE
[NPC] Aldor Marksman - Certain Aldor Marksmen at the Terrace of Light use Shopkeeper dialogue when clicked

### ISSUE THREAD

**Issue by Hidole555** [Atlantiss/NetherwingBugtracker#3653]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 
Three Aldor Marksmen (ID: 19, GUIDLow: 24149), (ID: 20, GUIDLow: 24193), (ID: 22, GUIDLow: 24206) have Shopkeeper dialogue when left-clicked by the player instead of the generic Draenei greeting.
Note that not all Marksmen have this problem, After seeing ID: 19, 20, and 22 having voice issues I tracked down Aldor Marksman (ID: 21, GUIDLOW: 24210) and found him to be working as intended.
**Current behaviour**:
When one of the three marksmen are clicked they respond with shopkeeper lines such as "I hope you find something useful!" and "You wish to buy, trade?"
**Expected behaviour**: 
They should give the generic Draenei greeting lines such aas "Good fortune!" and "The Legion's end draws near."
**Server Revision**: 
2497 [Branch: master, Hash: bd79729c02ee]

**Comment by Hidole555** [comment]
Appears fixed after the latest server restart.

**Comment by Hidole555** [comment]
Reopened because I was mistaken. I checked the Terrace of Light today and Aldor Marksmen are still receiving Shopkeeper dialogue. Currently in-game of the four I mentioned earlier, two of them are using proper dialogue and two of them are still using shopkeeper quotes. This extends to the Aldor Marksmen that are outside training as well.

I looked up the wowhead names for the sounds and it seems that there are two sounds currently used by Aldor Marksmen when clicked on, the first being DraeneiMaleNobleGreetings and the second being DraeneiMaleMilitaryVendor. It seems that whatever is responsible for assigning sound files to these NPCs has MilitaryVendor in it instead of MilitaryGreetings. The expected behavior is that they use DraeneiMaleMilitaryGreetings instead of DraeneiMaleMilitaryVendor.
